doc/specifications/proposed/content_download/content_download.md

```diff
@@ -85,13 +87,3 @@ would be redirected to
 ```
 /content/download/123/file_field/bc.pdf?version=6
 ```
-
-
-## Options
-
-### IgorwFileServeBundle
```

Comment:
Well, it was an option, but it wasn't implemented. At least what we have supports resuming.